.pre-commit-config.yaml

```diff
@@ -63,10 +63,10 @@ repos:
         args:
             - --license-filepath
             - license_header.txt
--   repo: https://gitlab.com/pycqa/flake8.git
-    rev: 3.9.2
+-   repo: https://github.com/pycqa/flake8.git
```

Comment:
just curious: why do we need this change?

Reply:
flake8 migrated from gitlab to github and was blocking merge gates this morning